__tests__/app.test.js

```diff
@@ -49,9 +49,27 @@ describe("app", () => {
             topic: "mitch",
             created_at: "2020-06-06T09:10:00.000Z",
             votes: res.body.article.votes,
+            comment_count: 2,
```

Comment:
My previous test failed, so I added comment_count property to previous test as well.